cmd/av/stack_submit.go

```diff
@@ -71,13 +72,17 @@ If the --current flag is given, this command will create pull requests up to the
 			result, err := actions.CreatePullRequest(
 				ctx, repo, client, tx,
 				actions.CreatePullRequestOpts{
-					BranchName: branchName,
-					Draft:      config.Av.PullRequest.Draft,
+					BranchName:    branchName,
+					Draft:         config.Av.PullRequest.Draft,
+					NoOpenBrowser: true,
 				},
 			)
 			if err != nil {
 				return err
 			}
+			if result.Created {
+				lastCreatedPullRequest = result.Branch.PullRequest
```

Comment:
An alternative behavior would be to wait until all the PRs are created first, and then bulk-open all new windows at once.

With stack visualizations in https://github.com/aviator-co/av/pull/264, IMO that's unnecessary since the last created PR will have links to all the other stacks.

However, if it's off perhaps it would still make sense to open a bunch of new windows, or alternatively just open the repo to the PR list, e.g., https://github.com/aviator-co/av/pulls.

Thoughts?

Reply:
@draftcode could you let me know what you think?

Reply:
I feel like if we open PR pages (which is default), then it's better to open all (i.e. changing this to bulk open). I usually do not use `av stack submit` (usually use `av pr create), but I want to check the newly created PRs for the last check.

Reply:
Or make this configurable.

Reply:
Alright, for now I updated to just bulk open them all at once at the end. That feels less disruptive than opening one by one and requiring you to keep switching between browser and terminal.